v2/charts/azure-service-operator/templates/secret.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.createAzureOperatorSecret -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aso-controller-settings
+  namespace: {{.Release.Namespace}}
+type: Opaque
+data:
+  AZURE_SUBSCRIPTION_ID: {{ .Values.azureSubscriptionID | b64enc | quote }}
+  AZURE_TENANT_ID: {{ .Values.azureTenantID | b64enc | quote }}
+  AZURE_CLOUD_ENV: {{ .Values.cloudEnvironment | b64enc | quote }}
+
+  {{- if .Values.azureClientID }}
+  AZURE_CLIENT_ID: {{ .Values.azureClientID | b64enc | quote }}
```

Comment:
Should also add an `AZURE_SYNC_PERIOD` option here and in the values.yaml

Reply:
Yep, also added - AZURE_OPERATOR_MODE and AZURE_TARGET_NAMESPACES